tsconfig.json

```diff
@@ -15,7 +15,8 @@
   "include": [
     "./packages/**/*",
     "./draft-packages/**/*",
-    "./legacy-packages/**/*"
+    "./legacy-packages/**/*",
+    "./storybook/**/*"
```

Comment:
Was storybook never included in our original tsconfig? interesting 🤔

Reply:
Yep :P Maybe it started not having any TS so was exempt. But then we started adding things, and I thought it was weird that I wasn't getting any help/errors.